oozie-to-airflow/README.md

```diff
@@ -321,28 +335,117 @@ From the [Oozie documentation](https://oozie.apache.org/docs/5.1.0/WorkflowFunct
 Currently the only supported way of configuring the map-reduce action is with the
 inline action configuration, i.e. using the `<configuration>` tag in the workflow's XML file definition.
 
-**4. Streaming and pipes**
+**3. Streaming and pipes**
 
 Streaming and pipes are currently not supported.
 
-### Spark Example
 
-The spark example, like the other examples can be run as:
+### Pig Example
+
+The Pig example can be run as:
```

Comment:
```suggestion
The Pig example can be converted as:
```

Reply:
I think "running" in the context of O2A means running the conversion.
What you are referring to is "running a system test", which is different in my opinion.

Besides this phrase was used before in the docs, so the original author must have thought it was appropriate.
I'd suggest leaving it as is.

Reply:
Running is fine.